torchrl/envs/common.py

```diff
@@ -551,6 +473,136 @@ def to(self, device: DEVICE_TYPING) -> _EnvClass:
         self.device = device
         return self
 
+class _EnvClass(_EnvBase):
```

Comment:
Can we remove the underscore? 
What is the difference with EnvBase?

Reply:
EnvBase is the common class to EnvMB and EnvStateful(renamed _EnvClass)

Reply:
got it
I think the docstrings should be super clear about what each of these can and cannot do